sorc/ncep_post.fd/ALLOCATE_ALL.f

```diff
@@ -335,6 +338,9 @@ SUBROUTINE ALLOCATE_ALL()
 !
 !     FROM VRBLS2D
 !
+      allocate(CAPE(im,jsta_2l:jend_2u))
+      allocate(CIN(im,jsta_2l:jend_2u))
+      allocate(APCP(im,jsta_2l:jend_2u))
```

Comment:
@SamuelTrahanNOAA Variables should be changed for 2D decomposition as:
allocate(CAPE(ista_2l:iend_2u, jsta_2l:jend_2u)

May I ask for the intention of adding these three new variables?

Reply:
Those store the three derived input variables to IFI. The "im" was a mistake; I forgot to update that when I merged the 2D decomposition changes.